.circleci/config.yml

```diff
@@ -89,26 +94,25 @@ exports: &exports
   run: |
     # It is not possible to resolve env vars in the environment section:
     # https://circleci.com/docs/2.0/env-vars/#using-bash_env-to-set-environment-variables
-    echo "export VERSION=${CIRCLE_TAG:-build-$CIRCLE_WORKFLOW_ID}" >> $BASH_ENV
+    # DEV_TAG and PROD_TAG are the tags used for the Kubeapps docker images
+    echo "export DEV_TAG=build-${CIRCLE_SHA1}" >> $BASH_ENV
+    echo "export PROD_TAG=${CIRCLE_TAG:-latest}" >> $BASH_ENV
+    # Apart than using a DEV_TAG we use a different image ID to avoid polluting the tag
```

Comment:
Apart than -> Apart from

Personally, I don't really care about polluting the tags, but I'll leave that for you to decide.

Reply:
I didn't thought about that but it's true that it helps to navigate through available tags. It also closer to official docker images (they don't pollute that with tags) and it's easy to maintain.